articles/virtual-machines/dedicated-hosts-how-to.md

```diff
@@ -64,6 +71,18 @@ Not all host SKUs are available in all regions, and availability zones. You can
 az vm list-skus -l eastus2  -r hostGroups/hosts  -o table
 ```
 
+You can also verify if a VM SKU supports Ultra disks.
+
+```azurecli-interactive
+subscription="<yourSubID>"
```

Comment:
What I've seen in the docs is using things like mySubId, myResourceGroup, myRegion, myVM for the pieces where customers should replace the code snippet info with their own. Something to consider for subscription, region, and vmSize in this snippet.

Reply:
this is a snippet I have directly copied from VM doc, but I can change it to 'my'